src/commands/command_utils.ts

```diff
@@ -0,0 +1,93 @@
+import { Uri, Diagnostic, Range, DiagnosticSeverity } from "vscode";
+import { NodeMetaData } from "../domain";
+import { DBTProject } from "../manifest/dbtProject";
+
+export function findModelProblems(
```

Comment:
This is not scalable. This should go in the respective class that does the documentation checks.